core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalSystem.java

```diff
@@ -168,7 +170,8 @@ public void waitForCatchup() {
           (int) Configuration.getMs(PropertyKey.MASTER_UFS_JOURNAL_MAX_CATCHUP_TIME))
           .setInterval(Constants.SECOND_MS));
     } catch (InterruptedException | TimeoutException e) {
-      LOG.info("Journal catchup is interrupted or timeout", e);
+      // TODO(jiacheng): I don't understand why we don't crash here
```

Comment:
not very sure but seems like this method is only called when this property is enabled : https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/conf/PropertyKey.java#L2701-L2712

it's more like a best effort than a must.

Reply:
According to our [code style](https://drive.google.com/file/d/1kWJnSFcnBWUdM5kJ7hrrPKYD64WYZR_5/view?usp=sharing), TODOs should not be left in the code.

Reply:
yea this TODO just meant I'd like to discuss with you guys, i will remove this